pinot-common/src/main/java/com/linkedin/pinot/common/segment/SegmentMetadata.java

```diff
@@ -143,6 +143,13 @@
   String getBitmapInvertedIndexFileName(String column, String segmentVersion);
 
   /**
+   * returns the version of the Pinot Hadoop jar that was used to create this segment
+   * @param column
+   * @return
+   */
+  String getPinotHadoopJarVersion();
```

Comment:
Consider more generic naming like getGeneratorName... it could be 'realtime' logic or, say, pinot-admin tool